modules/ROOT/pages/6.0-release-notes-premium-changes.adoc

```diff
@@ -45,6 +49,53 @@ No Premium Plugins released with {productname} 6.0 are backwards compatible with
 | `toc_header`                        | `tableofcontents_header`         | Option                              | NB: _Table of Contents_ is now a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].
 |===
 
+[[hindi-malay-and-vietnamese-translations-added-to-premium-plugins]]
+== Hindi, Malay, and Vietnamese translations added to Premium Plugins
+
+The following Premium Plugins now present in Hindi, Malay, and Vietnamese:
+
+* Accessibility Checker
+* Advanced Tables
+* Comments
+* Enhanced Image Editing
+* Export
+* PowerPaste
+* Real-time Collaboration (RTC)
+* Spell Checker Pro
+* Table of Contents
+
+// to do above
+// link out to each plugin page
+//
+// pre-requisite
+// xref or https as link syntax?
```

Comment:
xref links should be used within docs, unless we are linking outside of the docs section (https for tiny website links)

Reply:
xrefs added.